### DISCUSSION TITLE
Religion Q: "Buy best building first" target.

### DISCUSSION THREAD

**Discussion by mossball** [kitten-science/kitten-scientists#739] [Q&A] [answered]
I'm currently using the v2.0.0-beta.8, since that's what the description links to as the "stable version".
Do any of the newer versions display the next "target" for the "Buy best building unicorn first?".
I'd just like to know what it's saving up for.

I'm not sure how to add the newest version as a "Bookmarklet" otherwise I'd quickly test it myself in a incognito tab >_>

**Comment by oliversalzburg** [comment, marked as answer]
This doesn't exist even in the latest development version, but it's an excellent idea! If you want, you can raise it as a feature request, otherwise I will create the ticket later myself. Thanks for bringing it up. (#740)

When trying out "unstable" builds, be sure to have a backup of your game. In general, to try the latest nightly build with a bookmarklet, you should be able to use:

```
javascript:(function(){var d=document,s=d.createElement('script');s.src='https://kitten-science.com/nightly.js';d.body.appendChild(s);})();
```

Notice that this one says `nightly.js` in the URL.

**Reply by mossball** [reply]
Thanks for the bookmarklet! I'll give the newer versions a try for a while before deciding whether to add them to Tampermonkey permanently or not.

As for the feature request, I'd honestly appreciate it if you could open the ticket later, I think someone who's a bit more tech-savvy than me could format the ticket/request in the most readable way. :]
